Tribler/Main/tribler.py

```diff
@@ -27,7 +27,7 @@
     # in the windows and mac distribution, there may be no version available.
     # so select a version only when there is any available.
     if wxversion.getInstalled():
-        wxversion.select("2.8-unicode")
+        wxversion.select("3.0-osx_cocoa")
```

Comment:
True, not sure which wxversion I should select on Windows yet so I will change it after I know this.